py-polars/src/conversion.rs

```diff
@@ -125,10 +125,10 @@ impl<'a> FromPyObject<'a> for Wrap<BooleanChunked> {
 
 impl<'a> FromPyObject<'a> for Wrap<Utf8Chunked> {
     fn extract(obj: &'a PyAny) -> PyResult<Self> {
-        let (seq, len) = get_pyseq(obj)?;
+        let len = obj.len()?;
```

Comment:
```suggestion
        # TODO: extend `get_pyseq` for this case and replace
        # `obj.len()?` here by `get_pyseq(obj)?`.
        let len = obj.len()?;
```

Reply:
I would rather suggest to drop `get_pyseq` entirely. It is not clear which purposes it serves if the `PySequece` API is not used.